renku/cli/config.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017-2018 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Get and set Renku repository or global options.
+
+Set values
+~~~~~~~~~~
+
+You can set the registry URL to a custom value with:
```

Comment:
This doc string reads a bit weird -- it says above "Get and set repository or global options" then here it says "set the registry URL". I know that registry and repository aren't the same thing, but someone reading this might think they are being used as synonyms. It might be better to say something like "You can set various Renku configuration options, for example the image registry URL, with a command like: "